src/routes/v1/apps/handlers/deleteImage.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
 
                 await deleteMedia(media_uuid, db, transaction)
 
-                transaction.commit()
+                await transaction.commit()
```

Comment:
If we dont wait for the commit it would delete the files even if the db-transaction fails